src/pages/Main/List/ListMain.js

```diff
@@ -10,18 +11,24 @@ const ListMain = () => {
     fetch('./data/ListMock.json')
       .then(data => data.json())
       .then(data => {
-        setHouseList(data.data);
-        setLoading(false);
+        setTimeout(() => {
+          setHouseList(data.data);
+          setLoading(false);
+        }, 2000);
       });
   }, []);
 
-  if (loading) return;
-
   return (
     <S.CardWrapper>
-      {houseList?.map(data => (
-        <ListCard key={data.id} data={data} />
-      ))}
+      {loading ? (
+        <Skeleton width="316px" height="400px" number={20} />
+      ) : (
+        <>
+          {houseList.map(data => (
+            <ListCard key={data.id} data={data} />
+          ))}
+        </>
+      )}
```

Comment:
```suggestion
if(loading) return <Skeloeton width="316px" height="400px" number={20}>

return (<ul>{ houseList.map(......) } </ul>)
```

이렇게 ealry return 방식을 사용하면 가독성도 더 좋아질 것 같아요!

Reply:
넵 해당 사항 수정했습니다 감사합니다.